pipeline/stage.go

```diff
@@ -105,3 +108,51 @@ func (s *StageSlice) Sanitize(driver string) *StageSlice {
 		return nil
 	}
 }
+
+// Empty returns true if the provided stage is empty.
+func (s *Stage) Empty() bool {
+	// return true if the container is nil
```

Comment:
I think the comment here has a typo i.e. ~`container`~

```suggestion
	// return true if the stage is nil
```